run-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+ES_VER=0.90.13
+ES_TMP=/tmp/serviced_elastic
+ES_DIR=$ES_TMP/elasticsearch-$ES_VER
+
+if [ -e $ES_TMP/pid ]; then \
+	kill `cat $ES_TMP/pid`; \
+fi
+rm -rf $ES_TMP
```

Comment:
lines 7-10 can be wrapped into a function like `stop_elasticsearch`, then you can call that function before/after starting ES instead of repeating the code at lines 24-27